shared/yarn.lock

```diff
@@ -11625,9 +11625,9 @@ react-native-fast-image@5.1.1:
   resolved "https://registry.yarnpkg.com/react-native-fast-image/-/react-native-fast-image-5.1.1.tgz#2595323a926ade0b3a37bd627eb55e3ebf67598b"
   integrity sha512-kEzgZxbbXYhy27u5GnhrKitn+XDBFAHSDUJdYC6llMi5cDPjgcqhOAQABj0K+ga5pn+/xPZLmD882rrUGiwVVA==
 
-"react-native-image-picker@git://github.com/keybase/react-native-image-picker#v0.27.1-with-keybase-fixes":
+"react-native-image-picker@git://github.com/keybase/react-native-image-picker#akalin/DESKTOP-8743-image-capture-secure":
```

Comment:
will switch back to v0.27.1-with-keybase-fixes once https://github.com/keybase/react-native-image-picker/pull/6 is merged